docs/5.0/index.md

```diff
@@ -1,45 +1,59 @@
+description: How to install and quickly get up and running with Teleport to set up SSH and Kubernetes access to cloud environments.
 ---
 
 # Introduction
 
 ## What is Teleport?
 
-Gravitational Teleport is a gateway for managing access to clusters of Linux
-servers via SSH or the Kubernetes API. It is intended to be used instead of
-traditional OpenSSH for organizations that need to:
+Teleport is a Unified Access Plane for your infrastructure.
 
-* Secure their infrastructure and comply with security best-practices and
-  regulatory requirements.
-* Have complete visibility into activity happening across their infrastructure.
-* Reduce the operational overhead of privileged access management across both
-  traditional and cloud-native infrastructure.
+* Single sign-on access plane for all your SSH servers, Kubernetes, Databases and Web Apps.
+* List all environments and see which servers are online with a single CLI command or via a live view in a browser.
+* Customize access workflows and integrate access into existing DevOps tools or your own applications using your favorite programming language.
+* Define and enforce policy to handle idle sessions, concurrent sessions, and to share and record interactive sessions across all environments.
 
-=== "Teleport Demo"
+<iframe  width="712" height="400" src="https://www.youtube-nocookie.com/embed/0HlyGk8dihM?rel=0&modestbranding=1" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
 
-    <iframe  width="712" height="400" src="https://www.youtube-nocookie.com/embed/0HlyGk8dihM?rel=0&modestbranding=1" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
+## Why Use Teleport
 
-=== "Teleport Explainer Video"
+Here are some of the most popular use-cases for Teleport:
 
-    <iframe width="712" height="400" src="https://www.youtube-nocookie.com/embed/GvAD5aNcdBA?rel=0&modestbranding=1&widget_referrer=gravitational.com/teleport/docs" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowullScreen></iframe>
-
-Teleport aims to be a cloud-native SSH solution, i.e. it makes it natural to think of
-environments, not servers. Below is a list of the most popular Teleport features:
-
-* Single SSH/Kubernetes access gateway for an entire organization.
-* SSH certificate based authentication instead of static keys.
-* Avoid key distribution and [trust on first use](https://en.wikipedia.org/wiki/Trust_on_first_use) issues by using auto-expiring keys signed by a cluster certificate authority (CA).
-* Enforce 2nd factor authentication.
-* Connect to clusters located behind firewalls without direct Internet access via SSH bastions.
+* Use short lived certificates instead of static keys for SSH, Kubernetes, Databases and Web Apps.
+* Structured events and session recording/replay for `ssh` and `kubectl exec`.
```

Comment:
```suggestion
* Structured events and session recording/replay for `ssh` and `kubectl`.
```
^ Version 5 captures more of kubectl, extending our support beyond `kubectl exec`